.github/workflows/cpp-linter.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: cpp-linter/cpp-linter-action@master
+      - uses: cpp-linter/cpp-linter-action@composite-action
```

Comment:
You'll have to revert this before merging I think.

Reply:
Yes, we need to change the branch name to master when we are ready to merge, here is just for testing.

Reply:
I think we're ready to change this to `main`